tests/operator_test.go

```diff
@@ -1430,17 +1392,19 @@ var _ = Describe("ALL Operator tests", func() {
 	})
 })
 
-func getCDICrAndPods(f *framework.Framework) (*cdiv1.CDI, *corev1.PodList) {
+func getCDIPods(f *framework.Framework) *corev1.PodList {
 	cdiPods, err := f.K8sClient.CoreV1().Pods(f.CdiInstallNs).List(context.TODO(), metav1.ListOptions{})
 	Expect(err).ToNot(HaveOccurred(), "failed listing cdi pods")
 	Expect(len(cdiPods.Items)).To(BeNumerically(">", 0), "no cdi pods found")
+	return cdiPods
+}
 
+func getCDI(f *framework.Framework) *cdiv1.CDI {
+	By("Getting CDI resource")
 	cdis, err := f.CdiClient.CdiV1beta1().CDIs().List(context.TODO(), metav1.ListOptions{})
 	Expect(err).ToNot(HaveOccurred())
-	Expect(len(cdis.Items)).To(Equal(1))
-	cr := &cdis.Items[0]
-
-	return cr, cdiPods
+	Expect(cdis.Items).To(HaveLen(1))
```

Comment:
so do we prefer that test will fail because they expect to have at least 1 cdi? or should we skip if len < 1 like somewhat done before:
				if len(cdis.Items) < 1 {
					Skip("CDI CR does not exist.  Probably managed by another operator so skipping.")
				}

Reply:
why would you run CDI func tests if you have no CDI on the cluster? failing looks good to me...